python_bindings/bindings.cpp

```diff
@@ -212,13 +213,26 @@ class Index {
                 });
             };
             cur_l+=rows;
-
-
         }
+    }
 
+    std::vector<std::vector<dist_t> > GetDataReturnNumpy(py::object ids_ = py::none()) {
```

Comment:
This method returns a vector object which is later converted to a plain python list. Does not seem be an issue, since it is not performance critical, but the name of the method is a bit confusing (GetDataReturn*Numpy*).